functions.cmake

```diff
@@ -216,6 +216,11 @@ function(_add_executable)
       disable_pax_mprotect(${ARGV})
     else()
       add_executable(${ARGV})
+      list(GET ARGV 0 CLR_CMAKE_CURRENT_TARGET)
+      set_target_properties(
+        ${CLR_CMAKE_CURRENT_TARGET} 
+        PROPERTIES
+        VS_GLOBAL_CL_MPCount ${CLR_CMAKE_CPU_CORES})
```

Comment:
A nit - coding convention - the closing ) should be on a separate line aligned with the set_target_properties.

Reply:
Will fix